samples/powerpoint/shapes/get-set-shapes.yaml

```
@@ -0,0 +1,227 @@
+              rectangle.width = getRandomBetween(minNewShapeHeight, maxNewShapeHeight);
+              rectangle.left = getRandomBetween(0, slideWidth - rectangle.width);
+              rectangle.top = getRandomBetween(0, slideHeight - rectangle.height);
+              rectangle.fill.foregroundColor = generateRandomHexColor();
+            }
+            finalTable += "Done<br>";
+            $("#slide-tags").empty();
+            $("#slide-tags").append(finalTable);
+          });
+        }
+
+        let currentLeft = 0;
+        let currentTop = 0;
+        async function arrangeSelected() {
+          // Arranges the selected shapes in a line from left to right.
+          await PowerPoint.run(async (context) => {
+            const shapes = context.presentation.getSelectedShapes();
+            const shapeCount = shapes.getCount();
+            shapes.load("items");
+            await context.sync();
+            let maxHeight = 0;
+            shapes.items.map((shape) => {
+              shape.load("width,height");
+            });
+            await context.sync();
+            shapes.items.map((shape) => {
+              shape.left = currentLeft;
+              shape.top = currentTop;
+              currentLeft += shape.width;
+              if (shape.height > maxHeight) maxHeight = shape.height;
+            });
+            await context.sync();
+            currentLeft = 0;
+            if (currentTop > slideHeight - 200) currentTop = 0;
+          });
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+          try {
+            await callback();
+          } catch (error) {
+            // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+            console.error(error);
+          }
+        }
+    language: typescript
+template:
+    content: |
+        <section class="ms-font-m">
+            <p>This sample shows how to get selected shapes, and how to select specific shapes.</p>
+        </section>
+
+        <section class="samples ms-font-m">
+            <h3>Try it out</h3>
+        <button id="createShapes" class="ms-Button"><span class="ms-Button-label">Create shapes</span></button>
+        <br><button id="getSelectedShapes" class="ms-Button"><span class="ms-Button-label">Get selected shapes</span></button>
+        <br><button id="setSelectedShapes" class="ms-Button"><span class="ms-Button-label">Select first two shapes on slide one</span></button>
+        <br><button id="changeFill" class="ms-Button"><span class="ms-Button-label">Change fill color of selected shapes</span></button>
+        <br><button id="saveShapeSelection" class="ms-Button"><span class="ms-Button-label">Save selected shapes</span></button>
+        <br><button id="loadShapeSelection" class="ms-Button"><span class="ms-Button-label">Load selected shapes</span></button>
```

I think this button text should be: 
```suggestion
        <br><button id="loadShapeSelection" class="ms-Button"><span class="ms-Button-label">Load saved shapes</span></button>
```